tests/test_docs.py

```diff
@@ -235,13 +235,19 @@ def test_level_17_force_template_type_from_moban_file(self):
         folder = "level-17-force-template-type-from-moban-file"
         self._raw_moban(["moban"], folder, expected, "simple.file")
 
-    def test_level_18_user_defined_template_types(self):
+    def test_level_18_user_defined_template_types_a(self):
         from datetime import datetime
         expected = "{date}\n".format(date=datetime.now().strftime("%Y-%m-%d"))
 
         folder = "level-18-user-defined-template-types"
         self._raw_moban(["moban"], folder, expected, "a.output")
 
+    def test_level_18_user_defined_template_types_b(self):
```

Comment:
please refer to `def test_level_15_copy_templates_as_target`, where you will find answers to verify more than one file. It is optimal solution but is sufficent.